sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -304,6 +308,8 @@ def compute_table_name(row):
 NOTE: This job name template does not have backwards compatibility guarantees.
 """
 BQ_JOB_NAME_TEMPLATE = "beam_bq_job_{job_type}_{job_id}_{step_id}{random}"
+"""The number of shards per destination when writing via streaming inserts."""
+DEFAULT_SHARDS_PER_DESTINATION = 500
```

Comment:
I believe Java uses 50 shards. Do we need a larger default for Python ?

Reply:
we've been able to reach ~1k EPS per worker in Python. If we have 50 shards, we'll only reach ~50k maximum. I'd like to have a larger default, so we don't automatically cap EPS at a very low rate.

Reply:
Note that Java also allows users control this through a pipeline option and Dataflow support ask users to update this parameter as they fit.

Reply:
https://github.com/apache/beam/blob/master/sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryOptions.java#L58

Reply:
These flags for streaming inserts can be added in a separate PR though.

Reply:
Filed https://issues.apache.org/jira/browse/BEAM-10680 - thanks Cham!